extensions/wikia/TaskManager/SpecialTaskManager_body.php

```diff
@@ -44,17 +44,11 @@ public function  __construct() {
 	public function execute( $subpage ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This function "execute" has 166 lines, which is greater than the 150 lines authorized. Split it into smaller functions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS138)